packages/modals/src/styled/StyledBackdrop.ts

```diff
@@ -18,21 +16,42 @@ export interface IStyledBackdropProps {
   isAnimated?: boolean;
 }
 
+const animationName = keyframes`
+  0% {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
```

Comment:
Should we combine percentage and `from/to` syntax? Does `from` work?

Reply:
This was moved over from https://unpkg.com/browse/@zendeskgarden/css-modals@6.4.19/dist/index.css, but `from` would work too.

Reply:
It works. This is an effect of `postcss` compiling to the smallest representation. The original is `0%` - `100%`: https://github.com/zendeskgarden/css-components/blob/master/packages/modals/src/_keyframes.css